seb_openedx/seb_keys_sources.py

```diff
@@ -23,7 +23,11 @@ def from_other_course_settings(course_key):
     course_module = get_course_module(course_key, depth=0)
     if hasattr(course_module, 'other_course_settings'):
         other_settings = course_module.other_course_settings
-        return other_settings.get('SAFE_EXAM_BROWSER', None)
+        other_settings = other_settings.get('SAFE_EXAM_BROWSER', {})
+        if course_key.html_id() in other_settings:
```

Comment:
In the rest of the codebase we use `six.text_type(course_key)` to convert in a string type. Just for consistency, lets use that.